srcpkgs/wlroots/template

```diff
@@ -1,7 +1,8 @@
 # Template file for 'wlroots'
 pkgname=wlroots
-version=0.6.0
-revision=1
+version=0.5.0
+reverts="0.6.0_1"
```

Comment:
`This field must be defined before version and revision fields in order to work as expected.`